src/editorial-source-components/CustomDropdown.tsx

```diff
@@ -0,0 +1,63 @@
+import { css } from "@emotion/react";
+import { space } from "@guardian/src-foundations";
+import { Option, Select } from "@guardian/src-select";
+import type { Option as OptionValue } from "../plugin/types/Element";
+import { inputBorder } from "./inputBorder";
+import { labelStyles } from "./Label";
+
+const parentStyles = css`
+  width: initial;
+  div {
```

Comment:
Is this to style the wrapper around Select – and if so, would it be worth documenting that coupling with a comment to explain what we're doing? If the shape of Select changes and breaks this, for example, a comment might make that link clearer.

Reply:
Yes that's right - applying CSS to the `.svg` and the existing wrapper `div` wasn't possible in the existing Source Select component so I've added these styles to a new parent `div`.

I hadn't considered that the Select element might change in the future, especially as it's currently flagged as being 'in development' in Source. A comment (at the very least) would definitely be sensible.

Reply:
Yeah, there's a balance here – I'd almost be in favour of copy-pasting the element in its entirety, as at least then we absolutely know what we're getting – and I suspect it wouldn't be large! Comment if not sounds good though.

Reply:
That makes sense - though I think it would make sense to restructure the element a bit so as not to have to rely on indirect css rules (particularly the `!important` I had to add in.

Reply:
Having said that - the Select element from Source is in its own package so I think we'd only see problems with it if someone specifically bumped the version of `@guardian/src-select` and the interface changed.

I also checked Source and Select isn't marked as 'in development' after all, so I think it's unlikely the interface will change much.

Reply:
Let's use it as is for now. Should be easy to change if we need.

Reply:
@rhystmills to double check, did you agree that this still merited a comment?

Reply:
Yes - I thought I had added one already actually, will do so now!